jrnl/plugins/datecount_exporter.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+# encoding: utf-8
+# Copyright (C) 2012-2021 jrnl contributors
+# License: https://www.gnu.org/licenses/gpl-3.0.html
+
+from .text_exporter import TextExporter
+from .util import get_date_counts
+
+
+class DatecountExporter(TextExporter):
+    """This Exporter can lists the tags for entries and journals, exported as a plain text file."""
```

Comment:
Copy/paste error (still refers to tags plugin).

Reply:
Woops!

Reply:
Fixed!